tests/app/config/services.yaml

```
@@ -17,6 +17,14 @@ services:
             $populators:
                 - '@Neusta\ConverterBundle\Tests\Fixtures\Populator\PersonNamePopulator'
 
+    test.address.converter: &test-address-converter
```

This anchor isn't used anywhere

```suggestion
    test.address.converter:
```